pkg/config/apm.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+func setupAPMTelemetry(config Config) {
```

Comment:
I think all of this can go with the rest (just like the profiler and debugger). It doesn't warrant a separate function.